wcfsetup/install/files/style/layout/form.scss

```diff
@@ -225,6 +225,20 @@ html[data-color-scheme="dark"] .formError {
 	--color-error: #ff4747;
 }
 
+.formSuccess {
+	--color-success: rgb(46, 139, 87);
```

Comment:
This might require an additional declaration for the dark mode:
```css
html[data-color-scheme="dark"] .formSuccess {
    --color-success: rgb(some, color, here);
}
```